manager-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -15,7 +15,7 @@
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 use Symfony\Component\Config\Definition\ConfigurationInterface;
 
-final class Configuration implements ConfigurationInterface
+class Configuration implements ConfigurationInterface
```

Comment:
is this change related to the PR?

Reply:
Nope. Reverted in 1bf08f8fc15d2241edda342e57602b1b59a285e2.